### ISSUE TITLE
DOI of version 1.x of dataset points to GBIF page which displays data and doi of version 2.x

### ISSUE THREAD

**Issue by ymgan** [gbif/ipt#2096]
Hello,

I find it a little confusing when the DOI of version 1.x of a dataset points to the GBIF dataset page which is displaying latest version of data (in this case, we have version 2.x) and shows the DOI of version 2.x. We intentionally minted a new DOI for the major update to make a distinction between these two versions (1.x and 2.x) but this feels confusing to me.

I have 2 examples here:

IPT page | version 1 DOI | version 2 DOI | GBIF dataset page
---|---|---|---
https://ipt.biodiversity.aq/resource?r=belgica121 | https://doi.org/10.15468/56bv6z | https://doi.org/10.48361/kawuhz | https://www.gbif.org/dataset/b635be2e-76ea-4600-8f83-549601653c0a
https://ipt.biodiversity.aq/resource?r=mapppd_count_data | https://doi.org/10.15468/p1xkyb | https://doi.org/10.48361/zftxkr | https://www.gbif.org/dataset/f7c30fac-cf80-471f-8343-4ec5d8594661

Also, is it possible to show the version 1 DOI on the IPT page?

**Comment by ktotsum** [comment]
I assume this is related to the setting of preferred identifier and alternative identifiers, but not sure this can be done from IPT. 

@mike-podolskiy90 do you have any ideas?

**Comment by mike-podolskiy90** [comment]
Thanks @ymgan and @ktotsum 
@ymgan I'm not quite sure what would like to have? Initially your dataset didn't have a dedicated DOI, so GBIF assigned one and it point to the gbif.org page

**Comment by ymgan** [comment]
> Initially your dataset didn't have a dedicated DOI, so GBIF assigned one and it point to the gbif.org page

Yes indeed. According to the [dataset versioning policy](https://ipt.gbif.org/manual/en/ipt/latest/versioning):

> - A new major version is assigned to the dataset (a) the first time it’s published, or (b) **after it has been republished following one or more scientifically significant changes to the dataset.** The publisher must decide what constitutes a scientifically significant change (see definition below for help).

(b) is what happened to the datasets above.

> - A scientifically significant change (a) typically affects the majority of records in the dataset, and (b) could change the results of a scientific analysis using the dataset.

> - A new major version leads to the creation of a new DOI, whereas a new minor version does not.

which is why we minted a new DOI via the datacite account of our IPT. 

If someone cites the DOI for version 1.x of the dataset because they uses the data from version 1.x but the DOI is pointing to GBIF page which has data from version 2.x of the dataset (which has scientifically significant changes) 🤔 I feel that is a little misleading?

> - The DOI metadata should be as rich as possible, including where possible (a) alternate identifiers for the dataset, (b) relationships to other versions, (c) relationships to articles the dataset cites, (d) ORCID where contacts are listed, etc.

I am not sure how this can be done. I thought everything is automatic when we mint a new DOI via the IPT. I looked at our datacite account (d) was automatically populated but there seems to have no field for (a), (b) and (c)?

> I'm not quite sure what would like to have?

I am sorry if this is not constructive, because I don't know what I can have if GBIF does not support versioning ...

And maybe ... it is me who don't know what I was doing when I minted the new DOIs for the major update. Maybe can you please point me to an example dataset with major update(s) (hence more than 1 DOI for the dataset versions) or please guide me through the process to do this right? 

Thanks a lot!!

**Comment by ymgan** [comment]
Thank you very much @ahahn-gbif !! Yes, I agree with you! I appreciate the solution proposed even if it is not ideal~ It is already one step forward addressing the alternate identifiers!
